src/commands/update.rs

```diff
@@ -111,10 +120,12 @@ impl UpdateSubcommand {
             resolved_graph.activated.len() - 1
         ));
 
-        let dependency_changes =
-            generate_depedency_changes(&lockfile.as_ids().collect(), &resolved_graph.activated);
-
-        render_update_difference(&dependency_changes);
+        progress.enable_steady_tick(Duration::from_millis(100));
+        progress.suspend(|| {
+            let dependency_changes =
+                generate_depedency_changes(&lockfile.as_ids().collect(), &resolved_graph.activated);
+                render_update_difference(&dependency_changes);
```

Comment:
The formatter isn't happy with this!

```suggestion
            let dependency_changes =
                generate_depedency_changes(&lockfile.as_ids().collect(), &resolved_graph.activated);
            render_update_difference(&dependency_changes);
```